test/workers/test_virtual.py

```diff
@@ -25,6 +25,9 @@ def test_send_msg():
     # get pointer to local worker
     me = sy.torch.hook.local_worker
 
+    # pending time to simulate lantency (optional)
+    me.message_pending_time = 0.1
```

Comment:
As me is global, shouldn’t we set it back to zero at the end of the test?

Reply:
It should be set